From: Daiki Ueno <ueno@gnu.org>
Date: Mon, 10 Oct 2016 10:33:36 +0200
Subject: core: Don't report error on setting option 'allow-emacs-pinentry'.

Previously "OPTION allow-emacs-pinentry" returned an error if the Emacs
pinentry is not compiled in.  Since it is merely an option, it would
make more sense to just return OK.

Suggested-by: Werner Koch <wk@gnupg.org>
Signed-off-by: Daiki Ueno <ueno@gnu.org>
---
 pinentry/pinentry.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pinentry/pinentry.c b/pinentry/pinentry.c
index d3f5fab..d5d70e0 100644
--- a/pinentry/pinentry.c
+++ b/pinentry/pinentry.c
@@ -883,8 +883,6 @@ option_handler (assuan_context_t ctx, const char *key, const char *value)
     {
 #ifdef INSIDE_EMACS
       pinentry_enable_emacs_cmd_handler ();
-#else
-      return gpg_error (GPG_ERR_NOT_SUPPORTED);
 #endif
     }
   else if (!strcmp (key, "invisible-char"))
